src/PackagistClient.php

```diff
@@ -84,6 +85,94 @@ public function getStatistics(): ?array
         return $this->request('statistics.json');
     }
 
+    /**
+     * Get security vulnerability advisories for specific packages and/or which have been updated since some timestamp.
```

Comment:
We usually don't add such lengthy comments on how to use methods, but prefer methods that are obvious how you should use them. 

We also don't like passing booleans to methods as this might hurt readability on the caller's side. Could you split this function into multiple other ones in which the use is clear?

Reply:
This is exactly the sort of thing I would have loved to be pointed out originally, when I hadn't yet written tests for the functionality as it's currently written. :p
I'll make the change, but really would have been nice to point this out when I was asking for exactly this sort of feedback originally.

Reply:
Split into two separate methods